nexus/src/app/background/tasks/region_snapshot_replacement_step.rs

```diff
@@ -604,7 +604,7 @@ mod test {
                     gen: 0,
                     opts: CrucibleOpts {
                         id: Uuid::new_v4(),
-                        target: vec![snapshot_addr],
+                        target: vec![snapshot_addr.parse().unwrap()],
```

Comment:
nit, unimportant: could the test function just take `snapshot_addr` as a `SocketAddrV6`?

Reply:
Possibly! I think this code was added after I originally drafted this change, and this was a mindless build fix after I rebased. I'll give that a shot.

Reply:
Should be cleaned up in f0c692d.